pkg/serviceapi/handler_images.go

```diff
@@ -94,26 +95,31 @@ func image(w http.ResponseWriter, r *http.Request, runtime *libpod.Runtime) {
 	}
 	ctx := context.Background()
 
-	switch r.Method {
-	case "DELETE":
-		force := false
-		if len(r.Form.Get("force")) > 0 {
-			force, err = strconv.ParseBool(r.Form.Get("force"))
-			if err != nil {
-				Error(w, "Something went wrong.", http.StatusBadRequest, err)
-				return
-			}
-		}
-		r, err := runtime.RemoveImage(ctx, newImage, force)
+	force := false
+	if len(r.Form.Get("force")) > 0 {
+		force, err = strconv.ParseBool(r.Form.Get("force"))
 		if err != nil {
-			Error(w, "Something went wrong.", http.StatusInternalServerError, err)
+			Error(w, "Something went wrong.", http.StatusBadRequest, err)
 			return
 		}
-		// TODO
-		// This will need to be fixed for proper response, like Deleted: and Untagged:
-		w.(ServiceWriter).WriteJSON(http.StatusOK, r)
 	}
+	_, err = runtime.RemoveImage(ctx, newImage, force)
+	if err != nil {
+		Error(w, "Something went wrong.", http.StatusInternalServerError, err)
+		return
+	}
+	// TODO
+	// This will need to be fixed for proper response, like Deleted: and Untagged:
+	w.(ServiceWriter).WriteJSON(http.StatusOK, r)
```

Comment:
You are returning the request as the body to the client.  

You need something like:
[]struct{
Deleted string 'json:"deleted"'
}  

see createImage() return.